chain/consensus/hierarchical/actors/subnet/subnet_actor.go

```diff
@@ -64,12 +65,13 @@ func (a SubnetActor) State() cbor.Er {
 // ConstructParams specifies the configuration parameters for the
 // subnet actor constructor.
 type ConstructParams struct {
-	NetworkName   string                     // Name of the current network.
-	Name          string                     // Name for the subnet
-	Consensus     hierarchical.ConsensusType // Consensus for subnet.
-	MinMinerStake abi.TokenAmount            // MinStake to give miner rights
-	DelegMiner    address.Address            // Miner in delegated consensus
-	CheckPeriod   abi.ChainEpoch             // Checkpointing period.
+	NetworkName      string                     // Name of the current network.
+	Name             string                     // Name for the subnet
+	Consensus        hierarchical.ConsensusType // Consensus for subnet.
+	MinMinerStake    abi.TokenAmount            // MinStake to give miner rights
+	DelegMiner       address.Address            // Miner in delegated consensus
+	CheckPeriod      abi.ChainEpoch             // Checkpointing period.
+	ValidatorsNumber uint64                     // Number of validators.
```

Comment:
Would it be correct to say that this is not `ValidatorsNumber` but `MinValidators`, i.e. the minimum number of validators for the subnet to be able to start mining? (I know that this is a constant right now, but it'll change once we support reconfiguration, and that way we can leverage it for other consensus algorithms, and it allows us to even add this check for other consensus algorithms (like PoW)).

Happy to discuss sync if needed.

Reply:
fixed